envoy/formatter/substitution_formatter.h

```diff
@@ -1,112 +1,12 @@
 #pragma once
 
+#include "envoy/formatter/http_specific_formatter.h"
 #include "envoy/formatter/substitution_formatter_base.h"
 #include "envoy/http/header_map.h"
 
 namespace Envoy {
 namespace Formatter {
 
-/**
- * HTTP specific substitution formatter context for HTTP access logs or formatters.
- */
-class HttpFormatterContext {
```

Comment:
Please mention in the PR description that this class is moving.

Reply:
got it.